metadeploy/api/serializers.py

```diff
@@ -480,11 +493,22 @@ def _validate_results(self, data):
                 raise serializers.ValidationError(_("Invalid initial results."))
 
     def validate(self, data):
-        user = self._get_from_data_or_instance(data, "user")
-        org_id = self._get_from_data_or_instance(data, "org_id") or user.org_id
+        user = self._get_from_data_or_instance(data, "user", default=None)
+        org_id = self._get_from_data_or_instance(
+            data, "org_id", default=getattr(user, "org_id", None)
+        )
```

Comment:
@davisagli That's correct, but this is consistent with the existing functionality. If this `validate` is triggered by other unrelated changes to an existing Job, we want to use that job's `org_id` rather than overriding it. I think the way it's written we're not setting it unless it's missing?

Reply:
Oh, I see we are getting it from `data`, which is used on initial `create` when we pass it in from the `create_scratch_org` job... 🤔

Reply:
@jgerigmeyer Sounds like we just want `getattr(self.instance,"org_id, getattr(user, "org_id", None))`, and not the potential to receive a value from the request.

Reply:
@jgerigmeyer where? I see create_scratch_org creating a Job directly, not using the serializer.

Reply:
Yep; I missed that. I'll try it out 👍